src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/client/CommandCenter.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015 - 2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.ccbluex.liquidbounce.features.command.commands.client
+
+import net.ccbluex.liquidbounce.event.EventState
+import net.ccbluex.liquidbounce.event.Listenable
+import net.ccbluex.liquidbounce.event.events.PlayerNetworkMovementTickEvent
+import net.ccbluex.liquidbounce.event.handler
+import net.ccbluex.liquidbounce.features.command.Command
+import net.ccbluex.liquidbounce.features.command.builder.CommandBuilder
+import net.ccbluex.liquidbounce.utils.client.inGame
+import net.ccbluex.liquidbounce.utils.client.player
+import net.ccbluex.liquidbounce.utils.kotlin.EventPriorityConvention
+
+/**
+ * Center command
+ *
+ * Centers you at your current position.
+ */
+object CommandCenter : Listenable {
+
+    var center = false
```

Comment:
This variable implements a state-machine with two states. But it is not quite clear *what* both states are. You have to read the entire code *and* understand it in order to understand what `center = true` means.

I'd suggest to replace this boolean with an enum:
```kotlin
private enum HandlerState {
    INACTIVE,
    APPLY_ON_NEXT_EVENT
}
```